include/wx/filename.h

```diff
@@ -385,6 +385,15 @@ class WXDLLIMPEXP_BASE wxFileName
         { return Normalize(wxPATH_NORM_DOTS | wxPATH_NORM_ABSOLUTE |
                            wxPATH_NORM_TILDE, cwd, format); }
 
+        // Convenient helper for returning the absolute path corresponding to
+        // the given one.
+    wxString GetAbsolutePath(const wxString& cwd = wxEmptyString,
+                             wxPathFormat format = wxPATH_NATIVE) const
+        {
+            wxFileName fn(*this);
+            fn.MakeAbsolute(cwd, format);
```

Comment:
I'm surprised to find out `MakeAbsolute()` can't fail. So nothing to do here I suppose, as adding `(void)` here may look like it shouldn't be ignored, and in addition adding a note just here saying it can't fail doesn't do much. It just doesn't help if you know `MakeAbsolute()` returns bool and always had thought it was important to handle its failure.